lib/MusicBrainz/Server/WebService/Validator.pm

```diff
@@ -190,7 +190,7 @@ sub validate_inc
     my ($c, $version, $resource, $inc, $def) = @_;
 
     if (ref($inc)) {
-        $c->stash->{error} = 'Inc arguments must be combined with a space, but you provided multiple parameters';
+        $c->stash->{error} = 'Inc arguments must be combined with a space or plus sign, but you provided multiple parameters';
```

Comment:
Nitpicking here:

Technically the `+` is a space: in the form-urlencoded media type, spaces are encoded as `+`.  You can also use `%20`, but both represent a space.

So this now reads like "must be combined with a space or encoded space" which is kinda confusing since it always has to be encoded -- as either `+` or `%20`.

We could instead say something like "must be combined with a space (i.e. + or %20)," giving both encodings as examples, if that also helps?

Reply:
I guess I was thinking that the browsers automatically encode the spaces to `%20` before submission so you can just send an actual space, but I guess that's not true of all ways of hitting the WS maybe :) Added the specific wording instead.